src/app/view/components/_common/dot-global-message/dot-global-message.service.ts

```diff
@@ -0,0 +1,61 @@
+import { Injectable, OnInit } from '@angular/core';
+import { DotMessageService } from '../../../../api/services/dot-messages-service';
+import { DotEventsService } from '../../../../api/services/dot-events/dot-events.service';
+import { DotGlobalMessage } from '../../../../shared/models/dot-global-message/dot-global-message.model';
+
+
+
+/**
+ * Service to provide configurations for Global Messages.
+ * @export
+ * @class DotGlobalMessageService
+ */
+@Injectable()
+export class DotGlobalMessageService implements OnInit {
+    private config: DotGlobalMessage;
+
+    constructor(public dotMessageService: DotMessageService, private dotEventsService: DotEventsService) {}
+
+    ngOnInit(): void {
```

Comment:
Injectables classes doesn't have Lifecycle hooks, I don't think this is working correctly.

https://stackoverflow.com/questions/35110690/ngoninit-not-being-called-when-injectable-class-is-instantiated